src/paperqa/docs.py

```diff
@@ -780,6 +780,7 @@ async def aquery(
         summary_llm_model: LLMModel | None = None,
         embedding_model: EmbeddingModel | None = None,
         partitioning_fn: Callable[[Embeddable], int] | None = None,
+        response_schema: BaseModel | None = None,
```

Comment:
I think this should be `output_type: type[BaseModel] | TypeAdapter | JSONSchema | None = None`, are you sure it works?

Can you add or expand a unit test in `tests/test_paperqa.py` confirming this?